cypress/support/index.d.ts

```diff
@@ -1,11 +1,14 @@
-/* eslint-disable no-undef */
-
 declare namespace Cypress {
     interface Chainable {
         /**
          * Custom command to auth as a specific role service account
          * @example cy.auth('admin')
          */
         auth(roleName: string): Chainable<Element>
+        /**
+         * Custom command to initialize application configuration
+         * @example cy.initApplicationConfig()
+         */
+        initApplicationConfig(): Chainable<void>
```

Comment:
This prevents the Typescript compiler from complaining that the type is missing.